README.md

```diff
@@ -82,6 +84,14 @@ spec:
                   properties:
                     demo:
                       type: string
+
+  # Role holds an Aggregating Role used by the duck type to manage the ducks.
+	# If not specified, the Selectors are used to find a Role with an aggregation rule that matches a selector
```

Comment:
nit: tab

Reply:
the bot will patch this next round.
/lgtm

Reply:
neat, didn't know that